src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -165,6 +166,10 @@
      * The last job of the citizen.
      */
     private String lastJob                             = "";
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)